packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md

```diff
@@ -70,15 +97,15 @@ The client-based rate limit framework can operate in one of three modes:
 
 ### Check your rate limits with Okta Rate Limit headers
 
-The Rate Limit headers that are returned when the client-based rate limit is enabled are very similar to the headers that are returned through the [org-wide rate limits](/docs/reference/rl-best-practices/). The difference is that the header value is specific to a given client/IP/device combination rather than the org-wide rate limit values. Okta provides three headers in each response to report client-specific rate limits.
+The Rate Limit headers that are returned when the client-based rate limit is enabled are very similar to the headers that are returned through the [org-wide rate limits](/docs/reference/rl-best-practices/). The difference is that the header value is specific to a given client/IP/device identifier combination (for the `/authorize` endpoint) or IP/device identifier combination (for the `/login/login.htm` endpoint) rather than the org-wide rate limit values. Okta provides three headers in each response to report client-specific rate limits.
 
 > **Note:** If a client-based rate limit is in **Log per client** or **No action** mode, headers that are returned still reflect the org-wide rate limits.
 
 For client-specific rate limits, three headers show the limit that is being enforced, when it resets, and how close you are to hitting the limit:
```

Comment:
Would "For client-specific rates limits, the three headers show the ..." read better?